content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -534,18 +534,20 @@ please install corresponding CSI drivers.
 
 Older versions of Kubernetes also supported the following in-tree PersistentVolume types:
 
-* `photonPersistentDisk` - Photon controller persistent disk.
-  (**not available** starting v1.15)
-* `scaleIO` - ScaleIO volume.
-  (**not available** starting v1.21)
+* [`cephfs`](/docs/concepts/storage/volumes/#cephfs)
```

Comment:
is `cephfs` still intended to be listed on line 522? If so, could you also update the removal version there?

Reply:
removed.